### PR TITLE
Update pkijs

### PR DESCRIPTION
**Pull request by microshine**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by jucrouzet** [comment]
Fails due to #13831 merged, asked a revert

**Comment by andy-ms** [comment]
If you are linting, please keep the `tslint.json` file around as we use its presence to determine whether to lint a package.

File: pkijs/pkijs-tests.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-import Extension from "pkijs/src/Extension";
-import Attribute from "pkijs/src/Attribute";
-import SignedData from "pkijs/src/SignedData";
-import EncapsulatedContentInfo from "pkijs/src/EncapsulatedContentInfo";
-import SignerInfo from "pkijs/src/SignerInfo";
-import IssuerAndSerialNumber from "pkijs/src/IssuerAndSerialNumber";
-import SignedAndUnsignedAttributes from "pkijs/src/SignedAndUnsignedAttributes";
-import ContentInfo from "pkijs/src/ContentInfo";
+import { getCrypto, getAlgorithmParameters } from "pkijs";
+import { Certificate } from "pkijs";
```
Comment (andy-ms) [inline pkijs/pkijs-tests.ts]: Do you know that people should never use `pkijs/src/Certificate`? You could use [re-exports](http://www.typescriptlang.org/docs/handbook/modules.html) to support both, like `graphql` does.

Comment (microshine) [inline pkijs/pkijs-tests.ts]: I know about it. Thank you.

Some costomers want to use `rollup` for ES2015 (multi entrypoints). I'll think about it for next update

File: pkijs/index.d.ts [hunk 2]
```diff
@@ -1,16 +1,20 @@
-// Type definitions for pkijs
-// Project: https://github.com/PeculiarVentures/PKI.js
+// Type definitions for PKI.js API 2.0.48
```
Comment (mhegazy) [inline pkijs/index.d.ts]: A version should only have major.minor.

Comment (mhegazy) [inline pkijs/index.d.ts]: Adding a tslin.json should catch these issues.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by bowdenk7** [comment]
This PR has unaddressed comments and cannot be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**